content/chainguard/administration/cloudevents/image-copy-gcr/index.md

```diff
@@ -0,0 +1,196 @@
+
+To set up the sample application, you can create a Terraform configuration file and apply it to set up the necessary resources. To begin, create a new directory to hold the configuration and navigate into it.
+
+```sh
+mkdir ~/gcp-example && cd $_
+```
+
+After navigating into the new directory you can begin creating a Terraform configuration named `main.tf`.
+
+This configuration will consist of a single module. For the purposes of this example, we will call it `image-copy`. This module's `source` value will be the `iac` folder from the application code in the examples repository.
+
+```
+module "image-copy" {
+  source = "github.com/chainguard-dev/platform-examples//image-copy-gcp/iac"
+```
+
+The next five lines configure a few variables that you will need to update to reflect your own setup.
+
+* First, the configuration defines a `name` value. This will be used to prefix resources created by this sample application where possible.
+* Next, it specifies the GCP project ID where certain resources will reside, including the container image for this application (along with mirrored images), the Cloud Run service hosting the application, and the Service Account that authorizes pushes to the Google Artifact Registry.
+* Following that, the configuration specifies the Chainguard IAM organization from which we expect to receive events. This is used to authenticate that the Chainguard events are intended for you, and not another user. Images pushed to repositories under this organization will be mirrored to Artifact Registry.
+    * If you don't know your organization's UIDP, you can retrieve it by running `chainctl iam organizations list -o table`.
+* The next line specifies the location of the Artifact Registry repository and the Cloud Run subscriber.
+* The final line defines `dst_repo` value, which is used to create a name for the repository in the Artifact Registry where images will be mirrored.
+
+As an example, if the `name` value you specify is `chainguard-dev` and the `dst_repo` value is `mirrored` (as shown in the following example) any pushes to `cgr.dev/<organization>/foo` will be mirrored to `<location>-docker.pkg.dev/<project_id>/chainguard-dev-mirrored`
+
+Be sure to include to include a closing curly bracket after the final line.
+
+```
+  name = "chainguard-dev"
+
+  project_id = "<project-id>"
+
+  group = "<organization-id>"
+
+  location = "us-central1" 
+
+  dst_repo = "mirrored"
+}
+```
+
+You can create this file with a command like the following.
+
+```sh
+cat > main.tf <<EOF
+module "image-copy" {
+  source = "github.com/chainguard-dev/platform-examples//image-copy-gcp/iac"
+
+  name = "chainguard-dev"
+
+  project_id = "<project-id>"
+
+  group = "<organization-id>"
+
+  location = "us-central1" 
+
+  dst_repo = "mirrored"
+}
+EOF
+```
+
+After that you can apply the configuration to get the application up and running.
```

Comment:
```suggestion
Make sure to replace the placeholders with your own settings for `project_id` and `group`. In the next section, we'll see how to apply the configuration to get the application up and running.
```

Reply:
just an idea because when reading "after that you can apply"... I wasn't so sure I should run something on my own or wait for instructions

Reply:
Much clearer, thanks!